src/components/existing-request/existing-request-search.vue

```diff
@@ -122,6 +121,19 @@
         </v-form>
       </v-col>
 
+      <!-- Help Info -->
+      <v-col  v-if="isMobile" cols="12" md="3" lg="3" class="existing-request-info copy-small">
+        <div class="mx-8 my-10">
+          <ul>
+            <li>Check review time and approval status</li>
+            <li>Upgrade Name Request to Priority ($100.00)</li>
+            <li>Renew your Name Request for an additional 56 days before it expires ($30.00)</li>
+            <li>Cancel your Name Request</li>
+            <li>Edit details, download receipts, and more</li>
+          </ul>
+        </div>
```

Comment:
You could totally remove duplication here by changing it to a sub-component.

Reply:
Good idea

Reply:
Updated